target/linux/ath79/image/generic.mk

```diff
@@ -438,6 +438,16 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_cpe2g-9531
+  ATH_SOC := qca9531
+  DEVICE_TITLE = $$(DEVICE_VENDOR) $$(DEVICE_MODEL)$$(if $$(DEVICE_VARIANT), $$(DEVICE_VARIANT))
```

Comment:
The line with DEVICE_TITLE can be deleted, it's already been put in a general definition.